packages/calcite-components/src/components/color-picker-hex-input/color-picker-hex-input.e2e.ts

```diff
@@ -348,6 +348,14 @@ describe("calcite-color-picker-hex-input", () => {
           await assertTabAndEnterBehavior("", startingHex);
         });
 
+        it("commits hex chars when typing", async () => {
+          await selectText(input);
+          await page.keyboard.type("abcdef");
```

Comment:
Since committing on paste/input only works for longhand hex, can you assert on "abc" not changing the value, then assert the value changed once "def" is typed?

Reply:
I added assertion to only commit longhand hex values.